docs/DeveloperGuide.md

```diff
@@ -810,43 +800,134 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### 7.2 Adding a client
+
+1. Adding a client in the home window.
+
+   1. Test case: `add n/Steve Rogers p/91820392 e/thefirstavenger@test.com a/33 Apple Road, 928103`
+      Expected: New client with the details are added to the end of the list. Details of the added client are shown in
+                the status message and in the result display.
+   
+   2. Test case: `e/thefirstavenger@test.com add n/Steve Rogers a/33 Apple Road, 928103 p/91820392`
+      Expected: Similar to previous as the input order of the details does not matter.
+   
```

Comment:
this is wrong, command word should always be the first word. its the **order of parameters** that don't matter

Reply:
omg yes sorry sorry